packages/cli/src/utils/build.ts

```diff
@@ -1,15 +1,17 @@
 import { execa } from "execa";
 import { copyFileSync, mkdirSync, readdirSync, rmSync } from "fs";
 import path from "path";
+import { getOutDir } from "./forgeConfig";
 
-export async function forgeBuild(out = "out", options?: { clear?: boolean }) {
+export async function forgeBuild(options?: { clear?: boolean }) {
   if (options?.clear) {
+    const out = await getOutDir();
     console.log("Clearing forge build output directory", out);
     rmSync(out, { recursive: true, force: true });
```

Comment:
I think we can use `forge build --force` for this, rather than doing this manually.

<img width="433" alt="image" src="https://user-images.githubusercontent.com/508855/218577766-25dd3d9f-1309-415c-95ef-fe2e555f5ef9.png">

https://book.getfoundry.sh/reference/forge/forge-build

Reply:
Ahh, that's true. Might be worth asking the foundry team about the distinction/best practice. And maybe also an option to `forge build --clean` to clear output dir.